llvm/test/CodeGen/AIE/aie2/schedule/interblock/liveness-region.mir

```diff
@@ -0,0 +1,248 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+
+# RUN: llc --mtriple=aie2 -verify-machineinstrs --run-pass=postmisched --issue-limit=1 -debug-only=machine-scheduler %s -o - 2>%t.log
```

Comment:
Nit: I think we need the to run the liveregisters (or similar, I don't remember the name) to get verification of live ranges.  You'll probably need to add a couple of implicit-defs. But I think what is found now is correct.

Reply:
I could not find the required flag for verifying liveregisters. For now I will merge the changes